src/System.Private.CoreLib/shared/System/Text/Unicode/Utf8.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+
+namespace System.Text.Unicode
+{
+    /// <summary>
+    /// Provides facilities for inspecting, transcoding, and manipulating UTF-8 data.
+    /// </summary>
+    public static class Utf8
+    {
+        /// <summary>
+        /// Determines whether <paramref name="source"/> represents a well-formed UTF-8 sequence.
+        /// </summary>
+        /// <returns>
+        /// <see langword="true"/> if the sequence is well-formed UTF-8; <see langword="false"/> otherwise.
+        /// </returns>
+        /// <remarks>
+        /// Returns <see langword="true"/> if given an empty input.
+        /// </remarks>
+        public static bool IsWellFormed(ReadOnlySpan<byte> source)
```

Comment:
Consider adding `<param name="">` to the xml comments

Reply:
This API was removed.